website/source/docs/agent/configuration/index.html.md

```diff
@@ -0,0 +1,254 @@
+
+- `advertise` `(Advertise: see below)` - Specifies the advertise address for
+  individual network services. This can be used to advertise a different address
+  to the peers of a server or a client node to support more complex network
+  configurations such as NAT. This configuration is optional, and defaults to
+  the bind address of the specific network service if it is not provided.  If
+  provided, **all advertise addresses must be defined** (you cannot override
+  just one property).
+
+  - `http` - The address to advertise for the HTTP interface. This should be
+    reachable by all the nodes from which end users are going to use the Nomad
+    CLI tools.
+
+  - `rpc` - The address to advertise for the RPC interface. This address should
+    be reachable by all of the agents in the cluster.
+
+  - `serf` - The address advertised for the gossip layer. This address must be
+    reachable from all server nodes. It is not required that clients can reach
+    this address.
+
+    The default values are:
+
+    ```hcl
+    advertise {
+      http = "127.0.0.1:4646"
+      rpc  = "127.0.0.1:4647"
+      serf = "127.0.0.1:4648"
+    }
+    ```
+
+- `atlas` <code>([Atlas][atlas]: nil)</code> - Specifies if Nomad should connect
+  to Nomad Enterprise and Atlas.
+
+- `bind_addr` `(string: "127.0.0.1")` - Specifies which address the Nomad
+  agent should bind to for network services, including the HTTP interface as
+  well as the internal gossip protocol and RPC mechanism. This should be
+  specified in IP format, and can be used to easily bind all network services to
+  the same address. It is also possible to bind the individual services to
+  different addresses using the [addresses](#addresses) configuration option.
+
+- `client` <code>([Client][client]: nil)</code> - Specifies configuration which is specific to the Nomad client.
+
+- `consul` <code>([Consul][consul]: nil)</code> - Specifies configuration for
+  connecting to Consul.
+
+- `datacenter` `(string: "dc1")` - Specifies the data center of the local agent.
+  All members of a datacenter should share a local LAN connection.
+
+- `data_dir` `(string: required)` - Specifies a local directory used to store
+  agent state. Client nodes use this directory by default to store temporary
+  allocation data as well as cluster information. Server nodes use this
+  directory to store cluster state, including the replicated log and snapshot
+  data. This must be specified as an absolute path.
+
+- `disable_anonymous_signature` `(bool: false)` - Specifies if Nomad should
+  provide an anonymous signature for de-duplication with the update check.
+
+- `disable_update_check` `(bool: false)` - Specifies if Nomad should not check for updates and security bulletins.
+
+- `enable_debug` `(bool: false)` - Specifies if the debugging HTTP endpoints
+  should be enabled. These endpoints can be used with profiling tools to dump
+  diagnostic information about Nomad's internals. It is not recommended to leave
```

Comment:
This is copy-pasted from the old docs. If you want to change that, I'm happy to update it, but I did not write this.